RELEASE-NOTES.md

```diff
@@ -1,5 +1,40 @@
 # Prefect Release Notes
 
+## Release 2.16.1
+
+### TODO: !Highlight!
+
+### Enhancements
+- Add filter to prevent logging the current API key — https://github.com/PrefectHQ/prefect/pull/12072
+- Update `flow.serve` to support multiple schedules — https://github.com/PrefectHQ/prefect/pull/12107
+
+### Fixes
+- Clear runs even if deployment is paused, when updating/deleting schedules — https://github.com/PrefectHQ/prefect/pull/12089
+- Surface work pool not found error — https://github.com/PrefectHQ/prefect/pull/12087
+- Ignore `schedule` in `Deployment.build_from_flow` if it's out of date — https://github.com/PrefectHQ/prefect/pull/12100
+- Fix schedule instructions for `prefect deploy` — https://github.com/PrefectHQ/prefect/pull/12101
+- Fix reference to `prefect deployment schedule create` — https://github.com/PrefectHQ/prefect/pull/12117
+- Allow 500s on docker test `BuildError` — https://github.com/PrefectHQ/prefect/pull/12114
```

Comment:
this should have received a `maintenance` label, which would have filtered it out from being added here. we don't need to include any maintenance PRs

Reply:
(I also just retroactively added it to that PR for 🧹 )

Reply:
removed in [2ae8d53](https://github.com/PrefectHQ/prefect/pull/12120/commits/2ae8d531f7d7306970ac4efad8b6149f5bcefd91)